wikibaseintegrator/entities/baseentity.py

```diff
@@ -299,6 +303,23 @@ def get_entity_url(self, wikibase_url: str | None = None) -> str:
 
         raise ValueError('wikibase_url or entity ID is null.')
 
+    def validate_schema(self, entity_schema: str, language: str | None = None) -> bool:
```

Comment:
I think we should check the language code as well. And maybe rename to entity_schema_id to be extra clear?

Reply:
Or follow WDI and use eid, see https://github.com/SuLab/WikidataIntegrator/blob/main/wikidataintegrator/wdi_core.py#L209